test/integration/vpc_e2e_test.go

```diff
@@ -149,8 +163,44 @@ var _ = Describe(fmt.Sprintf("%s,%s: VPC Inventory", focusAws, focusAzure), func
 		err = verifyInventory("", true, false, true, "")
 		Expect(err).ToNot(HaveOccurred(), "timeout waiting to get VPC list")
 	})
+	It("CPA Add invalid/valid credentials", func() {
+		By("Add Secret with invalid credentials")
+		accountParams := cloudVPC.GetCloudAccountParameters("test-cloud-account"+namespace.Name, namespace.Name, true)
+		err := utils.AddSecret(kubeCtl, accountParams)
+		Expect(err).ToNot(HaveOccurred())
+		// Add CPA account.
+		err = utils.AddCloudAccount(kubeCtl, accountParams)
+		Expect(err).ToNot(HaveOccurred())
+
+		// Verify error is captured in CPA status field.
+		expectedError := ""
+		if accountParams.Provider == string(runtimev1alpha1.AWSCloudProvider) {
+			expectedError = "error describing ec2 vpcs"
```

Comment:
isnt just error presence sufficient? Pattern much is not required

Reply:
Thats sufficient enough. Updated the test.